test/service-worker/testCacheResources.js

```diff
@@ -29,14 +27,17 @@ describe('cacheResources()', function () {
 
     self.oghliner.RESOURCES = [];
     sinon.stub(Date, 'now').returns(testTimestamp);
-    sinon.stub(self, 'fetch').returns(Promise.resolve(mockedOkResponse));
-    sinon.stub(oghliner, 'prepareCache').returns(Promise.resolve(mockCache));
+    sinon.stub(self, 'fetch', function(request) {
```

Comment:
These changes are not related to this PR. Do you mind to drop them? We can discuss them in another PR.